Userland/Libraries/LibWeb/HTML/HTMLHyperlinkElementUtils.cpp

```diff
@@ -487,6 +487,10 @@ void HTMLHyperlinkElementUtils::follow_the_hyperlink(Optional<String> hyperlink_
     if (!url.is_valid())
         return;
 
+    // AD-HOC: LibWeb (and navigables) don't support help://. Once a "link click" event is sent to the chrome (i.e. the Help app) navigation is handled there.
+    if (url.scheme() == "help"sv)
+        return;
```

Comment:
should this be a FIXME for a more general mechanism that ideally has a callback to the embedding application? like the page_did_click_link before.

Reply:
Having looked at this again, I've instead implemented a sightly more general solution, that can fix more than just `help://` in #25857.